### ISSUE TITLE
Missing node_modules folder breaks --nofetch on cordova-cli 7.1.0

### ISSUE THREAD

**Issue by Chuckytuh** [apache/cordova-android#679]
# Bug Report

## Problem

Using cordova-cli 7.1.0 to try and create a new cordova project with the android platform, using --nofetch, causes the platform add to fail.

### What is expected to happen?
The expected behaviour is to successfully end up with a cordova project containing an android platform (8.0.0).

### What does actually happen?

Shortly after running `cordova platform add android --nofetch` the execution ends with the following error:

```
No scripts found for hook "before_platform_add".
Removing "cordova-" prefix from cordova-android
Adding android project...
Unable to load PlatformApi from platform. Error: Cannot find module 'q'
Error: Unhandled "error" event. (The platform "android" does not appear to be a valid cordova platform. It is missing API.js. android not supported.)
```

## Information

When running `cordova platform add android --nofetch` the execution path goes into [addHelper.downloadPlatform()](https://github.com/apache/cordova-lib/blob/rel/7.1.0/src/cordova/platform/addHelper.js#L292) with `opts.fetch = false` which ends up "downloading" the platform using [lazy_load.cordova_npm()](https://github.com/apache/cordova-lib/blob/rel/7.1.0/src/cordova/lazy_load.js#L113). 

Later, while attempting to load android's `Api.js` file from the cordova-android module, it fails because it's dependencies aren't installed, namely `Q` module.

This happens because cordova-android module is stored under npm_cache directory however, since `npm install` isn't executed, no dependencies aren't installed, hence the error.

In conclusion, it seems that cordova-android 8.0.0 is missing the node_modules folder that existed on previous versions! 

Updating cordova-cli is, sadly, not an option because cordova-fetch removed support for --nofetch and plugin installations are super slow.

### Command or Code

```bash
$ cordova create AppName com.some.package AppName
$ cd AppName && cordova platform add android --nofetch --verbose
```

### Environment, Platform, Device

Running on a macbook pro with OSX 10.14.

### Version information

* Host: osx 10.14
* cordova-cli: 7.1.4
* cordova-android: 8.0.0
* node: v8.11.3
* npm: 6.1.0

## Checklist

- [X] I searched for existing GitHub issues
- [ ] I updated all Cordova tooling to most recent version
- [X] I included all the necessary information above

**Comment by dpogue** [comment]
The new major versions of the platforms are not compatible with the old fetching (`--no-fetch`) mechanism. They require cordova-cli 8 as a minimum, and are ideally intended to work with the upcoming cordova-cli 9 (which also fixes the slow plugin installation problem).
